platform_it/pom.xml

```diff
@@ -1176,7 +1176,14 @@
                 <extraDependency3>splice_ee</extraDependency3>
             </properties>
         </profile>
-
+        <profile>
+            <id>dbaas</id>
+            <properties>
+                <extraDependency>splice_ee</extraDependency>
+                <extraDependency2>splice_ee</extraDependency2>
+                <extraDependency3>splice_ee</extraDependency3>
```

Comment:
nit: you may only guess how I (=a newcomer to the code) feel about `extraDependencyN` where this `N` says nothing about the purpose of the property. I'd consider it a naming bug.